daml/splice-dso-governance/daml/Splice/DsoRules.daml

```diff
@@ -1591,6 +1661,9 @@ executeActionRequiringConfirmation dso dsoRulesCid amuletRulesCid act = case act
       SRARC_CreateExternalPartyAmuletRules choiceArg -> void $ exercise dsoRulesCid choiceArg
       SRARC_CreateTransferCommandCounter choiceArg -> void $ exercise dsoRulesCid choiceArg
       SRARC_CreateUnallocatedUnclaimedActivityRecord choiceArg -> void $ exercise dsoRulesCid choiceArg
+      SRARC_ModifyValidatorLicenses modification -> case voteRequestCid of
+        None -> error "voteRequestCid is required for SRARC_ModifyValidatorLicenses"
+        Some voteRequest -> void $ exercise dsoRulesCid DsoRules_ModifyValidatorLicenses with modification; voteRequest
```

Comment:
Let's change EI to use `optVoteRequest : Optional AnyContract`, so we keep the ability to use instructions from the `Confirmation` workflow.

Reply:
Done